junit-jupiter-engine/src/test/java/org/junit/jupiter/engine/extension/ExtensionRegistryTests.java

```diff
@@ -36,6 +40,31 @@
 
 	private final ExtensionRegistry registry = createRegistryWithDefaultExtensions();
 
+	@BeforeEach
+	@AfterEach
+	void cleanProperties() {
+		System.clearProperty(EXTENSIONS_AUTODETECT_PROPERTY_NAME);
+	}
+
+	@Test
+	void newRegistryWithoutParentHasDefaultExtensions_usingServiceLocator() {
```

Comment:
I believe JUnit 5's tests do not by convention have an underscore (`_`) in their names. @sbrannen, is my observation correct?